monitors/SyntheticPegMonitor.js

```diff
@@ -54,7 +54,7 @@ class SyntheticPegMonitor {
         // `volatilityWindow`: Length of time (in seconds) to snapshot volatility.
         value: uniswapPriceFeed || medianizerPriceFeed ? 60 * 10 : 0, // 10 minutes
         isValid: x => {
-          return typeof x === "number" && x && x >= 0;
+          return typeof x === "number" && x >= 0;
```

Comment:
Unrelated to main PR changes. Having this first check previously prevented the `volatilityWindow` from taking on a value of 0.